### ISSUE TITLE
Save credentials on android

### ISSUE THREAD

**Issue by ghost** [jitsi/jitsi-meet#6584]
**Is your feature request related to a problem you are facing?**
Having to fill in the credentials everytime when connecting to a room is not very user friendly.

In the browser this is solved by the browser's save password feature, however on mobile nothing comparable seems to exist.

**Describe the solution you'd like**
It would be nice to be able to save the credentials for the server.

**Describe alternatives you've considered**
Filling in the credentials manually everytime when connecting to a room.

**Comment by nedimf** [comment]
Use ```SharedPreferences```.

**Comment by kksingla** [comment]
@saghul @nedimf how i can enable the user password for all users by default?? 

how i can limit number of users in 1 room?

**Comment by luixxiul** [comment]
@kksingla See my post: https://github.com/jitsi/jitsi-meet/issues/6610#issuecomment-629032160

**Comment by Medien-Treibhaus** [comment]
Is this in the works? It would really make life easier!

**Comment by saghul** [comment]
This is not being worked on right now, sorry.

**Comment by nedimf** [comment]
@Medien-Treibhaus still, you can use SharedPreferences for this. Before starting the session check if the user is logged in and then apply those credentials that have been stored.

**Comment by udopt** [comment]
Could you elaborate on how to use SharedPreferences for a saving credentials on an Android phone?

**Comment by blaueente** [comment]
Hey bot, just because nobody talked about this, this isn't getting less relevant. It is one thing when the team says they are not prioritizing this at the moment, or even adding "wontfix" themselves, but it's quite impolite to just try to sweep it under the rug like this.

**Comment by Medien-Treibhaus** [comment]
If our case study matters, I am the admin of a primary school and we are running an own jitsi instance. We protected it with credentials, so that only teachers can start a meeting. Having a strong password, it is a pain for my colleagues to type it to start a conference and the understandably complain about this.
Right now the only way I see to make it easier for them is compromising security with a shorter password or no credential use.
And as I'm already wishing, a dream would be if I could pack the credentials in a QR for them to scan with the app.
In any case thanks to the developers for their work!

**Comment by YoannPa** [comment]
> Could you elaborate on how to use SharedPreferences for a saving credentials on an Android phone?

I basically have the same question : how to use SharedPreferences within Jitsi Meet on Android ?

Appart from SharedPreferences, It would be nice to have the possibility somewhere to save several Jitsi server addresses, and the associated credentials for each of them.

Then you just have to click on the one you want and enter the virtual room name, and you are good to go :) !

**Comment by nedimf** [comment]
You can try save it SQLite or Room database.

**Comment by YoannPa** [comment]
> You can try save it SQLite or Room database.

How can I do that under Android from my smartphone ?

**Comment by blaueente** [comment]
> This issue has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs. Thank you for your contributions.

Just because there is no discussion it doesn't mean this issue is solved. Could someone please deactivate this bot?

**Comment by mgarmash** [comment]
I think this is important feature and should not be ignored.

**Comment by nedimf** [comment]
@saghul i wouldn't even know what exactly to contribute here, saving credentials is travel task on android platform

**Comment by saghul** [comment]
If it's so trivial send a PR and I'll be happy to review it.

**Comment by YoannPa** [comment]
@saghul I would have loved to contribute to that, but I have no idea how Jitsi code works.
Would it be possible to at least re-open the issue, in order that potential devs find it and contribute to it, and leave it open ?
That would be nice :) . Thank you.

**Comment by saghul** [comment]
I'd rather not. Nothing has happened in 2 years so there is no indication that anything new will happen.

Closed issues don't necessarily mean the idea is outright discarded, but it signals the interest. They can still be found and commented on.
